src/EventStore.Core/Index/IndexMap.cs

```diff
@@ -409,23 +463,64 @@ public void SaveToFile(string filename)
             Ensure.Nonnegative(commitCheckpoint, "commitCheckpoint");
 
             var tables = CopyFrom(_map);
-            AddTableToTables(tables, 0, tableToAdd);
+            AddTableToTables(tables, 0, tableToAdd); // only when is automatic merge
```

Comment:
This comment doesn't necessarily seem to be reflected in the code?

Reply:
place holder comment left over from when I was figuring out how to break it up. will delete